config/config.go

```diff
@@ -825,6 +829,26 @@ func convertToTerragruntConfig(
 	return terragruntConfig, nil
 }
 
+// Iterate over dependencies paths and check if directories exists, return error with all missing dependencies
+func validateDependencies(terragruntOptions *options.TerragruntOptions, dependencies *ModuleDependencies) error {
+	var missingDependencies []string
+	if dependencies == nil {
+		return nil
+	}
+	for _, dependencyPath := range dependencies.Paths {
+		fullPath := path.Join(terragruntOptions.WorkingDir, dependencyPath)
+		fileInfo, err := os.Stat(fullPath)
```

Comment:
NIT: you can use [`FileExists`](https://github.com/gruntwork-io/terragrunt/blob/master/util/file.go#L44).